Makefile

```diff
@@ -15,58 +13,58 @@ export GITHUB_PAGES_REPO ?= KohlsTechnology/eunomia
 export HELM_CHARTS_SOURCE ?= deploy/helm/eunomia-operator
 export HELM_CHART_DEST ?= $(GITHUB_PAGES_DIR)
 
-LDFLAGS := "-X github.com/KohlsTechnology/eunomia/version.Version=$(VERSION) \
-	-X github.com/KohlsTechnology/eunomia/version.Vcs=$(BUILD_COMMIT) \
+LDFLAGS := "-X github.com/KohlsTechnology/eunomia/version.Vcs=$(BUILD_COMMIT) \
 	-X github.com/KohlsTechnology/eunomia/version.Timestamp=$(BUILD_TIMESTAMP) \
 	-X github.com/KohlsTechnology/eunomia/version.Hostname=$(BUILD_HOSTNAME)"
 
-all: manager
+.PHONY: all build
 
-# Run tests
-native-test: generate fmt vet
-	go test ./pkg/... ./cmd/... -coverprofile cover.out
+all: build
 
-# Build manager binary
-manager: generate fmt vet
-	go build -o build/_output/bin/eunomia  -ldflags $(LDFLAGS) github.com/KohlsTechnology/eunomia/cmd/manager
+clean:
+	rm -rf build/_output
 
-# Build manager binary
-manager-osx: generate fmt vet
-	GOOS=darwin go build -o build/_output/bin/eunomia -ldflags $(LDFLAGS) github.com/KohlsTechnology/eunomia/cmd/manager
+# Build binary
+build:
```

Comment:
why remove the `generate` target instead of fixing the code by adding the missing directives: `//go:generate operator-sdk generate k8s` and `//go:generate operator-sdk generate openapi` in some good places?